MEDMI_Data_Download/met_extraction_script.py

```diff
@@ -483,7 +50,8 @@ def extract_data(self, date_range, latitude_range, longitude_range, outfile_suff
         for measurement in args.measurements:
             if measurement not in AVAILABLE_MEASUREMENTS:
                 raise ValueError('Unknown measurement: {}. Allowed measurements: {}'.format(measurement, AVAILABLE_MEASUREMENTS))
```

Comment:
When I check the help for this script I get these measurement options:

  --measurements M [M ...], -m M [M ...]
                        measurements to be extracted. Must be in (and defaults
                        to) [pollen, urtica, wind, ulmus, dewpoint, salix,
                        pressure, quercus, rel_hum, poaceae, temperature,
                        platanus, rain, fraxinus, corylus, betula, artemisia,
                        ambrosia, alnus]

Should we be listing all the pollen species separately as options here? (I think just having 'pollen' as an option would be better).

Or, if we do want to keep all pollen species as an option, can we move them to the end of the list, and add some text which makes it clear that selecting 'pollen' will select all the individual pollen species too?

Reply:
Have added this as issue, pasting your above comment.